src/cim/components/ClusterDeployment/ClusterDeploymentHostsDiscovery.tsx

```diff
@@ -30,7 +30,6 @@ const ClusterDeploymentHostsDiscovery: React.FC<ClusterDeploymentHostsDiscoveryP
   onSetInstallationDiskId,
   onSaveBMH,
   onSaveISOParams,
-  onFormSaveError,
```

Comment:
I noticed that in CIM, this prop never gets populated and I needed to make changes in the EditHostname component so I removed it. 

Please confirm it's working as expected.

Reply:
Good catch, the only use is following mock: 
https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/cim/EditAgentModal.tsx#L18

So by removing it here, we will need stolostron PR as well.

Reply:
I see. Given that the method is optional, the good thing is it can be changed in stolostron even before this code is approved.